src/components/Modals/Send/views/Address.tsx

```diff
@@ -129,15 +121,12 @@ export const Address = () => {
           <AddressInput
             rules={addressInputRules}
             enableQr={true}
-            placeholder={translate(
-              isYatSupported ? 'modals.send.addressInput' : 'modals.send.tokenAddress',
-            )}
+            placeholder={translate('modals.send.addressInput')}
```

Comment:
this is the translation that we want. no longer need `addressInput` (even though that is the better translation name if we want to update)
```suggestion
            placeholder={translate('modals.send.tokenAddress')}
```

Reply:
seems like we would still support ENS though, so maybe we want to update the translation just to remove `YAT`

Reply:
While this does look sane from a "just remove yat logic standpoint", this was previously wrong IMO, i.e:
https://github.com/shapeshift/web/blob/df13ac5de88d90ef7c29edd86998dca7d1e53a14/src/assets/translations/en/main.json#L1015
vs. https://github.com/shapeshift/web/blob/df13ac5de88d90ef7c29edd86998dca7d1e53a14/src/assets/translations/en/main.json#L1016

Reply:
Can follow up when removing yat validation entirely I guess since this is just banner work.

Reply:
Though reading into this more, I believe we should probably keep the `ethChainId` checks and render `addressInput` if true (we only support ENS resolution on ETH mainnet), else `tokenAddress` 👀

Reply:
Yeah, good catch-ception to find actual change we want haha

Reply:
ec01f00e5d3d415b1e6d30aafd9d5339b3d3f735